lib/screens/study_nurse_screen/configuration_screen/configuration_pages.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:picos/screens/study_nurse_screen/configuration_screen/pages/configuration_medication_and_therapy.dart';
+import 'package:picos/screens/study_nurse_screen/configuration_screen/pages/configuration_summary.dart';
+import 'package:picos/util/backend.dart';
+import 'package:picos/widgets/picos_ink_well_button.dart';
+
+import 'package:picos/themes/global_theme.dart';
+
+/// FormKey for the underlying PageView-Elements. 
+final GlobalKey<FormState> formKeyConfiguration = GlobalKey<FormState>();
+
+/// Stateful Widget for the PageView of configuration.
+class ConfigurationPages extends StatefulWidget {
+  /// Constructor of ConfigurationPages.
+  const ConfigurationPages({Key? key}) : super(key: key);
+
+  @override
+  State<ConfigurationPages> createState() => _ConfigurationPages();
+}
+
+class _ConfigurationPages extends State<ConfigurationPages> {
+  PageController controller = PageController();
+
+  final List<Widget> _list = <Widget>[
+    const ConfigurationForm(),
+    const ConfigurationVitalValues(),
+    const ConfigurationActivityAndRest(),
+    const ConfigurationBodyAndMind(),
+    const ConfigurationMedicationAndTherapy(),
+    const ConfigurationSummary()
+  ];
+
+  int _currentPage = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    final GlobalTheme theme = Theme.of(context).extension<GlobalTheme>()!;
+
+    return Scaffold(
+      backgroundColor: Colors.white,
+      appBar: AppBar(
+        title: Text(AppLocalizations.of(context)!.configuration),
+        backgroundColor: theme.darkGreen3,
+      ),
+      body: Form(
+        key: formKeyConfiguration,
+        child: PageView(
+          scrollDirection: Axis.vertical,
+          controller: controller,
+          onPageChanged: (int num) {
+            setState(() {
+              _currentPage = num;
+            });
+          },
+          children: _list,
+        ),
+      ),
+      bottomNavigationBar: Row(
+        children: <Widget>[
+          Expanded(
+            child: PicosInkWellButton(
+              text: AppLocalizations.of(context)!.back,
+              onTap: () {
+                if (_currentPage == 0) {
+                  Navigator.of(context).pop();
+                } else {
+                  controller.jumpToPage(_currentPage - 1);
+                }
+              },
+              buttonColor1: theme.grey3,
+              buttonColor2: theme.grey1,
+            ),
+          ),
+          Expanded(
+            child: PicosInkWellButton(
+              text: AppLocalizations.of(context)!.proceed,
+              onTap: () async {
+                if (_currentPage == _list.length - 1) {
+                  formKeyConfiguration.currentState!.save();
+                  bool backendCreatePatient = await Backend.createPatient();
+                  if (backendCreatePatient) {
+                    if (!mounted) return;
+                    Navigator.of(context)
+                        .pushNamed('/configuration-finish-screen');
```

Comment:
```suggestion
                    Navigator.of(context)
                        .pushReplacementNamed('/configuration-finish-screen');
```
As far as I can see the previous screen isn't needed anymore, so why pushing on top of the stack?